docs/data-cookbooks/browse-generation.md

```diff
@@ -338,7 +366,7 @@ Then click the gear in the upper right corner and click "ReRun":
 
 ![Screenshot of a rule edit page in the Cumulus dashboard with a pop-up menu open](../../assets/browse_processing_2.png)
 
-Tab over to executions and you should see the ```DiscoverGranulesBrowseExample``` workflow fire, succeed and then moments later the ```CookbookBrowseExample```.
+Tab over to executions and you should see the `DiscoverGranulesBrowseExample` workflow fire, succeed and then moments later the `CookbookBrowseExample`.
```

Comment:
```suggestion
Tab over to executions and you should see the `DiscoverGranulesBrowseExample` workflow run, succeed, and then moments later the `CookbookBrowseExample`.
```

What does "and then moments later the `CookbookBrowseExample`" mean? It seems like there's a word missing here.

Reply:
Probably in the original text.  It's suggesting that the first workflow will fire, then the second.